docs/integrations/pytorch_lightning.rst

```diff
@@ -22,7 +22,7 @@ PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI resea
 
 .. note::
 
-    This integration is tested with ``pytorch-lightning==1.0.0`` and current latest, and ``neptune-client==0.4.123`` and current latest.
+    This integration is tested with ``pytorch-lightning==1.0.7``, and ``neptune-client==0.4.123``.
```

Comment:
I think (if the examples didn't fail) it works with neptune-client==0.4.126

Reply:
yep, corrected.